README.md

```diff
@@ -166,3 +166,19 @@ the "greenlit" state at all. Resize operations that move into state "failed" or
 [pq-uri]: https://www.postgresql.org/docs/9.6/static/libpq-connect.html#LIBPQ-CONNSTRING
 [os-env]: https://docs.openstack.org/python-openstackclient/latest/cli/man/openstack.html
 [os-pol]: https://docs.openstack.org/oslo.policy/latest/admin/policy-json-file.html
+
+## Support, Feedback, Contributing
```

Comment:
We'll fix this ourselves but the short links summary should stay at the end.

Reply:
Fixed with the update as discussed.